src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.html

```diff
@@ -25,11 +25,11 @@ <h3 class="title">
 
 <div class="cards">
   <ix-dataset-details-card [dataset]="dataset"></ix-dataset-details-card>
-  <ix-permissions-card [dataset]="dataset" *ngIf="hasPermissions"></ix-permissions-card>
+  <ix-permissions-card [dataset]="dataset" *ngIf="hasPermissions && !dataset.locked"></ix-permissions-card>
   <ix-zfs-encryption-card
```

Comment:
I believe the button to unlock is in the encryption-card.

Reply:
Fixed